x-pack/plugins/apm/public/components/app/Main/index.js

```diff
@@ -25,17 +26,19 @@ export default function Main() {
       <UpdateBreadcrumbs />
       <Route component={ConnectRouterToRedux} />
       <Route component={ScrollToTopOnPathChange} />
-      {routes.map((route, i) => {
-        return route.switchRoutes ? (
-          <Switch key={i}>
-            {route.switchRoutes.map((route, i) => (
-              <Route key={i} {...route} />
-            ))}
-          </Switch>
-        ) : (
-          <Route key={i} {...route} />
-        );
-      })}
+      <AccessControl>
```

Comment:
This means that we block the UI until we have made the license and APM Server checks. Currently these checks do not run in parallel, which causes the delay to be even longer. Do you think this is a problem?